src/react/components/common/errorHandler/errorHandler.tsx

```diff
@@ -103,29 +102,40 @@ export class ErrorHandler extends React.Component<IErrorHandlerProps> {
                 message: reason.message,
                 title: reason.name,
             };
+        } else {
+            appError = {
```

Comment:
I think we can combine this with the previous case.  sometimes it happens that type Error has no message or name